pkgs/by-name/ya/yalc/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  nodejs,
+  yarnBuildHook,
+  yarnConfigHook,
+  yarnInstallHook,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "yalc";
+  version = "1.0.0-pre.54-unstable-2023-07-04";
```

Comment:
Where did this version number come from? It looks like we should just use `0-unstable-2023-07-04`, but I could be wrong.

Reply:
Comes from the package.json, but probably a good idea to use 0-unstable so that the unstableGitUpdater script can be used to auto-update it(not that I think it will be, last commit was over a year ago